src/shared/logToSentry.ts

```diff
@@ -0,0 +1,26 @@
+import { z } from 'zod';
+// other details like timestamp, and hostname/ IP address should be included by Sentry
+const zLogObject = z.object({
+    message: z.string(),
+    email: z.string().email().optional(),
+    data: z.object({}).optional()
+})
+
+enum logLevel{
+    ERROR,
+    INFO
+}
+
+export type logObject = z.TypeOf<typeof zLogObject>;
+
+const logToSentry = (tag: keyof typeof logLevel, logDetail: logObject) => {
+    if ((tag) === 'ERROR') {
+        console.error('LOG', tag, JSON.stringify(logDetail));
```

Comment:
I don't understand why you need the logToSentry function. Can't we just use console.log instead whereever logToSentry is called?

Reply:
In addition to what I have sent on WeChat, I was also thinking of keeping the logs in a consistent format and easier to manage